python_cross_site_scripting/QGB_QPSU/HTML.py

```
@@ -0,0 +1,2126 @@
+		
+		
+def markdown(response,t=''):
+	'''
+pip install markdown	
+	'''
+	if not t:
+		t=U.cbg()
+	# U,T,N	
+	import markdown
+	html=markdown.markdown(t,extensions=['fenced_code'])
+	response.headers['Content-Type']='text/html;charset=utf-8';
```

Ruff (0.13.3)

Statement ends with an unnecessary semicolon

Remove unnecessary semicolon

(E703)